### DISCUSSION TITLE
Script not working in Forge, please help

### DISCUSSION THREAD

**Discussion by Sucarip** [thundaga/SD-webui-txt2img-script#30] [Q&A] [answered]
Hi, I discovered this fantastic script for processing batches of images with hires.fix through a reddit thread ([here](https://github.com/thundaga/SD-webui-txt2img-script)) and I think it is exactly what I needed! :D, Unfortunately, I tried to test it on Forge and it doesn't work, it gives a Call Error:

![screenshot_725](https://github.com/user-attachments/assets/a806beaa-af7d-4275-9c23-7ce73c33b7be)

I have no knowledge of programming and no idea how to solve it, but it would be truly great to be able to use this script! It would be awesome if someone could help me, I think this script is exactly what I was looking for and it would help me a lot to be able to use it and add it to my workflow. 

Thank you very much in advance :)

**Comment by thundaga** [comment, marked as answer]
@Sucarip 
This error was raised previously please reference the issue here -> https://github.com/thundaga/SD-webui-txt2img-script/issues/25
For a resolution.

**Comment by Sucarip** [comment]
Thank you very much for make me notice it and direct me to the thread, I will check it ;)

**Comment by Sucarip** [comment]
Thank you very much, I have already tweaked the code and verified that it works. It's fantastic, exactly what I was looking for, thank you very much for your work and for sharing it.

If you are still working on the script, as a recommendation, it would be great if you could also add some boxes to modify the text of the negative prompt and be able to add and remove words (if it already exists, I haven't been able to find it).

Thank you again and congratulations on this awesome script ;D
